### ISSUE TITLE
Async

### ISSUE THREAD

**Issue by Rxann** [CodeWithSwastik/vscode-ext#6]
Hi, I am wondering if there are plans or things in the works to allow this library to use asynchronous functions etc. If not I think this would be great to add

**Comment by CodeWithSwastik** [comment]
Yes I am planning to shift to async soon. It should probably be async when 1.4.0 releases.

**Comment by CodeWithSwastik** [comment]
On second thought, the way vscode-ext is implemented right now we don't need the python code to be async since if the user uses a second command without finishing the previous command, it will spawn a new python process in order to run that. So technically its already async. 

If I transition to a handling it a bit differently we might need the python code to be async. So for now the python code will be synchronous.

**Comment by CodeWithSwastik** [comment]
v2.0.0 is now async so I'll close this issue now.
